src/cc_toolchain/cc_toolchain_config.bzl

```diff
@@ -200,6 +200,20 @@ def _make_common_features(ctx):
         ],
     )
 
+    result["colour_feature"] = feature(
+        name = "colour",
+        flag_sets = [
+            flag_set(
+                actions = [ACTION_NAMES.c_compile, ACTION_NAMES.cpp_compile],
+                flag_groups = [
+                    flag_group(
+                        flags = ["-fdiagnostics-color=always"],
+                    ),
+                ],
+            ),
+        ],
+    )
```

Comment:
Could we add a quick comment to the start of this group indicating what it's doing?

Reply:
done